packages/postgres-database/src/simcore_postgres_database/utils_users.py

```diff
@@ -18,17 +20,17 @@ class UserNotFoundInRepoError(BaseUserRepoError):
 
 class UsersRepo:
     @staticmethod
-    async def get_role(conn: SAConnection, user_id: int) -> UserRole:
+    async def get_role(conn: SAConnection, user_id: int) -> Any:
```

Comment:
why did you put Any here? instead of UserRole?

Reply:
I forgot the `Any` type here, but what I intended was to have `str` returned, which is the type stored in the table